x-pack/test/api_integration/apis/metrics_ui/index.js

```diff
@@ -16,7 +16,6 @@ export default function ({ loadTestFile }) {
     loadTestFile(require.resolve('./metrics'));
     loadTestFile(require.resolve('./sources'));
     loadTestFile(require.resolve('./snapshot'));
-    loadTestFile(require.resolve('./log_item'));
```

Comment:
We have a unit test for the search strategy now. The `data` plugin's `search` API should be tested by the `data` plugin.